### PR TITLE
Announce Kill

### PR DESCRIPTION
**Pull request by DevMatheuss**

### CODE REVIEW

File: _submissions/lua_script/Announcekill.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Script info
+
+Name: AnnunceKill
```
Comment (BarbzYHOOL) [inline _submissions/lua_script/Announcekill.md]:
```suggestion
Name: AnnounceKill
```

File: _submissions/lua_script/Announcekill.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+Author: Roberty
+Repository: https://github.com/merdlin/ElunasScripts/blob/master/killanuncio.lua
```
Comment (BarbzYHOOL) [inline _submissions/lua_script/Announcekill.md]: script name is not in english :(

File: _submissions/lua_script/Announcekill.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+Download: https://github.com/merdlin/ElunasScripts/archive/master.zip
+License: AGPL
+Script integration
+Includes configuration (.ext)?: No, configuration is in the script.
+Includes SQL patches?: No.
+Description
+the script announces the death of a player in a battle
```
Comment (BarbzYHOOL) [inline _submissions/lua_script/Announcekill.md]: What do you mean? it announces to every other players on the server?

File: _submissions/lua_script/Announcekill.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+Need help?
```
Comment (BarbzYHOOL) [inline _submissions/lua_script/Announcekill.md]: Remove these parts plz

File: _submissions/lua_script/Announcekill.md (hunk 3 above)
Comment (DevMatheuss) [reply _submissions/lua_script/Announcekill.md]: yes

**Comment by DevMatheuss** [comment]
ready I made all the requested changes

**Comment by BarbzYHOOL** [comment]
Hey, does it announce the death of player in pvp, pve or everything?

Because it's not clear by the name or description

**Comment by Helias** [comment]
@BarbzYHOOL is right. Please @merdlin add more information about the lua script.
Anyway I already forked the repo to add it in the catalogue, add a file icon.png to show it better in the catalogue.

Use this: https://github.com/azerothcore/lua-announce-kill

**Comment by BarbzYHOOL** [comment]
helias, i explained in discord why to not fork repos now :'(

**Comment by DevMatheuss** [comment]
> @BarbzYHOOL is right. Please @merdlin add more information about the lua script.
> Anyway I already forked the repo to add it in the catalogue, add a file icon.png to show it better in the catalogue.
> 
> Use this: https://github.com/azerothcore/lua-announce-kill

Where can I add icon.png?

**Comment by Helias** [comment]
Sorry, we decided to do not fork this, just add the links here merging this PR.
Also because this script it's really really similar to https://github.com/CoronaCore/LuaScripts/blob/master/lua_scripts/custom/pvp_announcer.lua

Probably we need to mention also Saj as author (?)

**Pull request merged**